jsonrpc/namespaces/eth/filters/filter_system.go

```diff
@@ -0,0 +1,340 @@
+// Copyright 2021 Evmos Foundation
+// This file is part of Evmos' Ethermint library.
+//
+// The Ethermint library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
```

Comment:
use code under the GNU license, there may be a requirement that the distribution must also follow the GNU license. However, since minievm currently uses a [non-GNU license](https://github.com/initia-labs/minievm?tab=License-1-ov-file#readme), it is important to clearly state in the PR description that GNU code from evmos/ethermint is being used. This will help the repository maintainer to be aware of this.

Reply:
@dongsam 
I have added `License issue` section to the PR description.

> This PR includes some codes that are derived from the Ethermint, which is licensed under the GNU Lesser General Public License (LGPL) v3. Minievm uses the Business Source License (BSL) 1.1, this could potentially lead to a license conflict.

Reply:
This can be problem hmm, I think we can rewrite filter logics to do not use comet but directly subscribe from indexer. 

I will change this part after merging it.